mod_reforged/hooks/config/strings.nut

```diff
@@ -935,11 +935,12 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
 	RF_ConcussiveStrikes = ::UPD.getDescription({
  		Fluff = "A strike to the head from this character means goodnight!",
+ 		Requirement = "Mace and Blunt Damage"
```

Comment:
`Attack` instead of `Damage` seems more fitting as we do not care about the amount or type of damage inflicted in any way. Especially in a world where MSU introduced a system where the same attack can randomly do either one or the other damage type if multiple are present.

Reply:
We are using "Damage" in all other cases so I went with that. But yes Attack is more accurate for how the perk is coded. I guess we should change this in other cases as well to "Attack".